README.md

```diff
@@ -173,6 +172,121 @@ On Ubuntu/Debian:
 
 	aptitude install doxygen graphviz
 
+#Dependencies on Windows
+
+Compilation under Windows is pretty complicated. It requires
+MSYS2 development environment to be installed. In order to install it follow
+the guide under following link:
+
+	https://msys2.github.io/
+
+Once it is installed open migw32 or mingw64 shell and install additional MSYS2
+tools:
+
+	pacman -S --noconfirm make git wget libtool autoconf automake doxygen \
+		gengetopt pkg-config patch
+
+and following packages for x86 version (mingw32) to build 32-bit binary:
+
+	pacman -S --noconfirm mingw-w64-i686-toolchain mingw-w64-i686-glib2 \
+		mingw-w64-i686-libnice mingw-w64-i686-dlfcn \
+		mingw-w64-i686-libwebsockets mingw-w64-i686-opus \
+		mingw-w64-i686-libogg mingw-w64-i686-jansson \
+		mingw-w64-i686-curl mingw-w64-i686-cmake mingw-w64-i686-ffmpeg
+
+Similarly you can install packages for x86_64 version (mingw64) to build 64-bit
+binary:
+
+	pacman -S --noconfirm mingw-w64-x86_64-toolchain mingw-w64-x86_64-glib2 \
+		mingw-w64-x86_64-libnice  mingw-w64-x86_64-dlfcn \
+		mingw-w64-x86_64-libwebsockets mingw-w64-x86_64-opus \
+		mingw-w64-x86_64-libogg mingw-w64-x86_64-jansson \
+		mingw-w64-x86_64-curl mingw-w64-x86_64-cmake mingw-w64-x86_64-ffmpeg
+
+There are some dependencies that you have to compile and install manually
+because its binaries are not accessible from pacman repository yet.
+However pacman scripts with all the patches for all remianing dependencies
+were submitted to
+
+	https://github.com/Alexpux/MINGW-packages
+
+for your convenience.
+
+libmicrohttpd (x86, x86_64):
+
+	mkdir mingw-w64-libmicrohttpd
+	cd mingw-w64-libmicrohttpd
+	wget https://raw.githubusercontent.com/Alexpux/MINGW-packages/master/mingw-w64-libmicrohttpd/PKGBUILD
+	makepkg-mingw -sLfi --noconfirm
+
+Sofia-SIP (x86, x86_64):
+
+	mkdir mingw-w64-sofia-sip-git
+	cd mingw-w64-sofia-sip-git
```

Comment:
Hi,
Well, Yes, Actually this is the shortest and the most comfortable way to have it compiled. Once binary packages are avaialble in pacman repository it would need simply run "pacman -S --noconfirm mingw-w64-i686-sofia-sip-git"
Hope that helps.
Best Regards